ext/data/templates/anki-field-templates-upgrade-v36.handlebars

```diff
@@ -0,0 +1,13 @@
+{{<<<<<<<}}
+{{#*inline "onyomi"}}
+    {{~#each definition.onyomi}}{{.}}{{#unless @last}}, {{/unless}}{{/each~}}
+{{/inline}}
+{{=======}}
+{{#*inline "onyomi"}}
+    {{~#each definition.onyomi}}{{.}}{{#unless @last}}, {{/unless}}{{/each~}}
+{{/inline}}
+
+{{#*inline "onyomi-hiragana"}}
+    {{~#each definition.onyomi}}{{hiragana (.)}}{{#unless @last}}, {{/unless}}{{/each~}}
+{{/inline}}
+{{>>>>>>>}}
```

Comment:
```suggestion
{{#*inline "onyomi-hiragana"}}
    {{~#each definition.onyomi}}{{hiragana (.)}}{{#unless @last}}, {{/unless}}{{/each~}}
{{/inline}}
```
When adding new handlebars, I don't think we should use the find-and-replace mechanism. Though we did it this way for handlebars updates 35 and 29, I think we should go back to the way it was done in 24 and earlier.